content/blog/gifts-q3-2020.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Gifts Q3 2020"
+date = 2020-09-22
+description = "Twice a year we distribute address space to those that have made valuable contributions to Urbit. Now called our *Gifts* program, the gifting of address space has been part of Urbit long before we had a grants program."
```

Comment:
Doesn't quite work as markup — 

<img width="584" alt="image" src="https://user-images.githubusercontent.com/20846414/94074872-196bc080-fdc8-11ea-9c49-2908e85abe68.png">

Reply:
I'll shorten it up a bit, ty! Checking out the rest of your suggestions momentarily.

Reply:
I mean, length is your preference, I just mean that it doesn't make it italic, heh